src/components/ListView.js

```diff
@@ -219,12 +221,14 @@ const ListViewRow = ({ card, versions, versionsLoading, checked, onCheck, addAle
       const { target } = event;
       const { name, value, tagName } = target;
 
+      const _value = name === 'cmc' ? parseFloat(value) : value;
```

Comment:
Design-wise, I'm not a fan of referring to specific target names in this method. It might not matter in a scenario as specific as this, but a better solution might be to either move the parsing logic to `syncCard` or find a way to send numerical values straight from the input.

Reply:
I agree wholeheartedly, and if y'all want to ignore this and fix it the right way, I'm totally fine with that. I was aiming for incremental correctness (and solving for my particular pain point 🤣) in this PR.

Reply:
My Mongo instance got blown away, so I'd have to rebuild it in order to make this type of change. If you're willing to accept this PR as a targeted fix, I'd appreciate it.